pkgs/development/python-modules/snuggs/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, lib, fetchPypi, click, numpy, pyparsing }:
+
+buildPythonPackage rec {
+  pname = "snuggs";
+  version = "1.4.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wg18xjmszifh3198dx3y3l31kjiaz9snsag529ih6iyr0pihmld";
+  };
+
```

Comment:
This test wasn't being ran https://github.com/mapbox/snuggs/blob/master/test_snuggs.py

Reply:
Fetched from github and added